### PR TITLE
Problem: can't customize flags of cli commands other than start

### PR DESCRIPTION
**Pull request by yihuang**
Solution:
- add map config `cmd-flags` to provide custom flags for all commands

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 ## UNRELEASED
 - [#52](https://github.com/crypto-com/pystarport/pull/52) support jsonnet as config language
 - [#56](https://github.com/crypto-com/pystarport/pull/56) Support override config.toml for all validators
+- [#]() Add config item `cmd-flags` to supply custom flags for all chain binary commands other than `start`, which is
```
Comment (yihuang) [inline CHANGELOG.md:6]:
```suggestion
- [#70](https://github.com/crypto-com/pystarport/pull/70) Add config item `cmd-flags` to supply custom flags for all chain binary commands other than `start`, which is
```

File: pystarport/cluster.py [hunk 2]
```diff
@@ -728,6 +728,7 @@ def create_account(cli, account, use_ledger=False):
             val["moniker"],
             chain_id=config["chain_id"],
             home=home_dir(data_dir, i),
+            **config.get("init-flags", {}),
```
Comment (yihuang) [inline pystarport/cluster.py:731]:
```suggestion
            **config.get("cmd-flags", {}),
```

File: pystarport/cluster.py [hunk 3]
```diff
@@ -809,6 +810,7 @@ def create_account(cli, account, use_ledger=False):
                 i=i,
                 min_self_delegation=node.get("min_self_delegation", 1),
                 pubkey=node.get("pubkey"),
+                **config.get("init-flags", {}),
```
Comment (yihuang) [inline pystarport/cluster.py:813]:
```suggestion
                **config.get("cmd-flags", {}),
```

File: pystarport/cluster.py [hunk 4]
```diff
@@ -867,7 +869,7 @@ def create_account(cli, account, use_ledger=False):
     # because the new binary may be a breaking one.
     doc = tomlkit.parse((data_dir / "node0/config/config.toml").read_text())
     if not doc["statesync"]["enable"]:
-        cli.validate_genesis()
+        cli.validate_genesis(config.get("init-flags", {}))
```
Comment (yihuang) [inline pystarport/cluster.py:872]:
```suggestion
        cli.validate_genesis(config.get("cmd-flags", {}))
```

**Review by mmsqe — COMMENTED** [review]
maybe could make `start-flags` as default, just add overwrite `cmd-flags`

**Comment by yihuang** [comment]
> maybe could make `start-flags` as default, just add overwrite `cmd-flags`

`start-flags` is specific for `start` cmd as the name suggests, `cmd-flags` apply to all cmds, and `start` cmd will have both.

**Review by mmsqe — APPROVED** [review]

**Pull request merged**